common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -110,7 +110,6 @@ function (HTML5Video, Resizer) {
         state.videoPlayer.currentTime = 0;
 
         state.videoPlayer.goToStartTime = true;
-        state.videoPlayer.stopAtEndTime = true;
```

Comment:
Oh, and I think I slightly misread; I see it set to false..

But if all of the logic for `stopAtEndTime` still works, I'm inclined to think we may as well leave it in place (I haven't inspected it closely yet).